pkg/ibmvpc/examples/out/explain_out/LBToResIPNode_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,296 @@
+Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca to 192.168.32.5 (iks-clusterid:1[192.168.32.5]) within ky-test-vpc
+=================================================================================================================================================
+
+No connections are allowed from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.36.6] to iks-clusterid:1[192.168.32.5];
+connection is blocked by Load Balancer
+
+Load Balancer: kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LoadBalancer] blocks connection to iks-clusterid:1[192.168.32.5] which is not one of its pool members
```

Comment:
In the case where the source is a LB and the destination is not a pool member I would print a message along the lines of 
```
The load balancer will not connect to this endpoint since it is not a pool member
```
and nothing else (drop the explanation about SGs and nACLs). I doubt users are interested in an analysis of the nACLs and SGs in this case. 

Either way, if you choose to elaborate SGs and nACLs on the way, do not use the phrasing "blocks connection" because the LB is not a firewall and cannot actually block connectivity.